corehq/messaging/scheduling/static/scheduling/js/conditional_alert_list.js

```diff
@@ -1,14 +1,13 @@
 hqDefine("scheduling/js/conditional_alert_list", function() {
     $(function() {
         var conditonal_alert_list_url = hqImport("hqwebapp/js/initial_page_data").reverse("conditional_alert_list");
-        var loader_src = hqImport("hqwebapp/js/initial_page_data").get("loader_src");
```

Comment:
You can also delete `loader_src` from https://github.com/dimagi/commcare-hq/blob/master/corehq/messaging/scheduling/templates/scheduling/conditional_alert_list.html#L12

Reply:
thanks https://github.com/dimagi/commcare-hq/pull/19027/commits/05b4e1d137dee24aa748282e0871aaa75dad4fa9